app/src/prod/kotlin.com.wire.android/ExternalLoggerManager.kt

```diff
@@ -0,0 +1,3 @@
+package com.wire.android
+
+fun initDatadogLogger(context: Context) {}
```

Comment:
to avoid detekt empty function block
```suggestion
fun initDatadogLogger(context: Context) = Unit
```